app/assets/stylesheets/main.css.scss

```diff
@@ -52,3 +52,6 @@ $compatibility-high-color: #84C93F;
 @import 'styles/pages/home';
 @import 'styles/pages/styles';
 
+// Import modular CSS
+@import "modules/**/**/*";
+@import "modules/**/**/**/*";
```

Comment:
AFAIK `/**/` means any directory _at that level_, and `*` means any file. You can add file extensions too I think `*.scss` https://github.com/chriseppstein/sass-globbing#stylesheet-syntax